numpy/core/fromnumeric.py

```diff
@@ -1774,7 +1774,8 @@ def nonzero(a):
         transpose(nonzero(a))
 
     The result of this is always a 2-D array, with a row for
-    each non-zero element.
+    each non-zero element. If there is concern the input may be a 0-d array,
+    use `nonzero(atleast_1d(a))` instead.
```

Comment:
I don't think we should be mentioning this next to the `transpose(nonzero(a))` example, I think the change in #13610 does a better job of that section.

Reply:
I don't think 0-d arrays are important enough to justify their own section here, but we should have a passing mention for them somewhere.

Reply:
The positioning of this statement is particularly confusing because it seems to suggest that `nonzero(atleast_1d(x))` is a substitute for `transpose(nonzero(x))`